prow/gerrit/client/syncer.go

```diff
@@ -37,40 +37,46 @@ type opener interface {
 }
 
 type SyncTime struct {
-	val    LastSyncState
-	lock   sync.RWMutex
-	path   string
-	opener opener
-	ctx    context.Context
+	val            LastSyncState
+	lock           sync.RWMutex
+	path           string
+	opener         opener
+	ctx            context.Context
+	stateChan      chan LastSyncState
+	forceFlushChan chan bool
+	uploadInterval time.Duration
 }
 
 func NewSyncTime(path string, opener opener, ctx context.Context) *SyncTime {
 	return &SyncTime{
-		path:   path,
-		opener: opener,
-		ctx:    ctx,
+		path:           path,
+		opener:         opener,
+		ctx:            ctx,
+		forceFlushChan: make(chan bool),
+		uploadInterval: time.Minute,
```

Comment:
To clarify, `time.Minute` is already a constant but I was wondering if we wanted something more to-the-point, such as `MAX_UPLOAD_INTERVAL` or something. This is more of a question and not a request to change anything.

Reply:
it would make sense to do so if we decide to turn this into a knob that could be adjusted by command line args, not seeing the needs now so would prefer to leave it as is